grpclb/build.gradle

```diff
@@ -11,17 +11,17 @@ description = "gRPC: GRPCLB LoadBalancer plugin"
 evaluationDependsOn(project(':grpc-core').path)
 
 dependencies {
-    compile project(':grpc-core'),
+    compileOnly libraries.javax_annotation
+    implementation project(':grpc-core'),
             project(':grpc-protobuf'),
             project(':grpc-stub'),
```

Comment:
I'm surprised this trailing comma doesn't cause parsing problems. (Travis isn't failing, so it seems it really isn't a problem, but it makes me wonder.)